developers/vscode/README.md

```diff
@@ -0,0 +1,39 @@
+# Ocean Protocol VSCode Extension
+
+The Ocean Protocol VSCode extension enables you to run compute jobs seamlessly within Visual Studio Code using Ocean Protocol's secure compute-to-data network. Execute your algorithms (JavaScript or Python) on data without ever exposing the raw data.
+
+## Getting Started
+
+Once installed, the extension adds an Ocean Protocol section to your VSCode workspace. Here you can configure your compute settings and run compute jobs using the currently active algorithm file.
+
+### Optional Setup
```

Comment:
Let's remove the "Optional" here, keep the setup and have a mention where the user can continue with defaults if necessary.

Reply:
But why remove "optional"? None of this setup is necessary, so it's helpful to let people know they can ignore this section if they want

Reply:
I wanted to replace with something else, Optional Setup can be put at the end, at a appendix section, since it's optional

Reply:
I don't agree with this. It makes sense to keep all of the UI related things together. Although it's optional, it is still more important than the development and contributing section which is relevant to much less people.

Reply:
I missed some updates, the order should be better now